### DISCUSSION TITLE
Rights Data in Arctos

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#4972] [General]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Is your feature request related to a problem? Please describe.**
There is a need for a central repository for recording rights information within Arctos. This is very important information for an art collection to capture and due to the complexity of this information, it may be best to create a rights module in Arctos. Much as was suggested in [creating a condition reporting module in Arctos](https://github.com/ArctosDB/arctos/issues/1908#issuecomment-573099209), we would then be able to link the information in a rights module with loans as a means of documenting image use requests and projects/publications for documenting use.

**Describe the solution you'd like**
Rights information to track in a rights module
Copyright status of the work (e.g., copyrighted, public domain, copyrighted orphan work). Controlled entry from authoritative sources: 
Creative Commons
RightsStatements.org
Traditional Knowledge 
Copyright holder (name of person or organization) and contact information
Copyright notice (exact wording to accompany reproductions of the work)
Publication status (i.e., has it been published, and if so, where, and when)
Summary and highlights of current copyright research or restrictions (e.g., is a license on file, are there any additional considerations such as privacy or publicity rights, other potential claimants, and where to go for more details and information). Further considerations include rights of indigenous peoples, contractual restrictions, and sensitive content and restricted materials (including obscenity, violence, and graphic depictions).
Whether non-exclusive license agreement is on file or not
Media attachment option for attaching licenses or other rights/permissions documents. 
Date of last copyright research and/or update and agent who completed that research and/or update

It is also critical to be able to clearly and concisely communicate some rights information to the public. The copyright of an artwork is evaluated at the artwork level, so there would need to be a way to link rights data from the module with an individual catalog record. 

Rights information to display for the public in catalog records
Copyright status of the work (e.g., copyrighted, public domain, copyrighted orphan work)
Copyright holder (name of person or organization)
Copyright notice (exact wording to accompany reproductions of the work)

**Describe alternatives you've considered**

**Additional context**
[Google Doc
](https://docs.google.com/document/d/1xgL-reugemAR0MOQapbPF4O_vikU2AUK94EzSA466fk/edit?usp=sharing)

**Priority**
Please assign a priority-label.

**Comment by campmlc** [comment]
This could potentially extend to a  permits module? We need to capture similar info for genomic material subject to the Nagoya Protocol. Borrowers of material need to be able to ascertain whether sample collection and loan follows the access and benefits sharing requirements of Nagoya.  

We have discussed having private media storage for permits. I assume this rights module would also require private media storage?

**Comment by dustymc** [comment]
I think it's the other way around - permits are a subset of "rights" - but yes, I think permit consolidation is part of what we'd hope to accomplish.

I think access is a separate discussion - that wouldn't change any functionality that I can see.
